DSCResources/MSFT_xDnsServerADZone/MSFT_xDnsServerADZone.schema.mof

```diff
@@ -8,4 +8,5 @@ class MSFT_xDnsServerADZone : OMI_BaseResource
     [Write, Description("DNS Server name")] String ComputerName;
     [Write, Description("Credential used to set zone"), EmbeddedInstance("MSFT_Credential")] String Credential;
     [Write, Description("Whether the DNS zone should be available or removed"), ValueMap{"Present","Absent"}, Values{"Present","Absent"}] String Ensure;
+    [Read] String CimSession;
```

Comment:
Change looks good.  Could you please add a test for this.

Reply:
I aggree with @iainbrighton I don't think this should be exposed externally and is an internal implementation.